edgecop-rspec.yml

```diff
@@ -21,3 +21,9 @@ RSpec/MultipleExpectations:
 
 RSpec/LetSetup:
   Enabled: false
+
+RSpec/ExampleLength:
+  Max: 8
```

Comment:
> Are we going to have the RSpec/ExampleLength cop enabled?

@andrew-edgescan, I bumped that up to 8 as we already had that rule in place for EdgeRails request specs. This might be controversial change though, so anyone who disagrees with this, please 🔥 me in the comments below, I'm ready 😎